ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -53,6 +53,42 @@ open class BottomCommandingController: UIViewController {
         }
     }
 
+    /// Indicates if the bottom commanding UI is hidden
+    ///
+    /// Changes to this property are animated.
+    @objc open var isHidden: Bool = false {
+        didSet {
+            if oldValue != isHidden && isViewLoaded {
+                if isInSheetMode {
+                    bottomSheetController?.isHidden = isHidden
+                } else if let bottomBarView = bottomBarView,
+                          let bottomConstraint = bottomBarViewBottomConstraint {
+                    if let animator = bottomBarHidingAnimator {
+                        animator.stopAnimation(true)
+                    }
+
+                    let springParams = UISpringTimingParameters(dampingRatio: Constants.BottomBar.hidingSpringDamping)
+                    let newAnimator = UIViewPropertyAnimator(duration: Constants.BottomBar.hidingSpringDuration, timingParameters: springParams)
+                    if isHidden {
+                        bottomConstraint.constant = -Constants.BottomBar.hiddenBottomOffset
+                        newAnimator.addCompletion { _ in
+                            bottomBarView.isHidden = true
+                        }
+                    } else {
+                        bottomBarView.isHidden = false
+                        bottomConstraint.constant = -Constants.BottomBar.bottomOffset
+                    }
+                    newAnimator.addAnimations {
+                        self.view.layoutIfNeeded()
```

Comment:
weak self?

Reply:
Good catch. Retain cycle in all of the addAnimation and addCompletion handlers. I added weak self to all of those.